blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_fr.properties

```diff
@@ -0,0 +1,92 @@
+EmptyState.pr=# Je pousse, tu tires\nLorsqu'une demande de tirage est ouverte pour le d\u00e9p\u00f4t _{0}_, Jenkins la testera et rapportera l'\u00e9tat de vos changements sur la demande de tirage sur GitHub.
+EmptyState.tests=Il n'y a aucune ex\u00e9cution de tests pour cette construction.\n\n
+Enable=Activer
+# home page / dashboard
+home.header.button.createpipeline=Nouveau Pipeline
+home.header.dashboard=Tableau de bord
+home.pipelineslist.header.branches=Branches
+home.pipelineslist.header.health=Sant\u00e9
+home.pipelineslist.header.name=Nom
+home.pipelineslist.header.pullrequests=Demandes de tirage
+home.pipelineslist.row.failing={0} en \u00e9chec
+home.pipelineslist.row.passing={0} avec succ\u00e8s
+Open.dashboard=Ouvrir le tableau de bord
+# pipeline details
+pipelinedetail.activity.button.run=Lancer maintenant
+pipelinedetail.activity.header.branch=Branche
+pipelinedetail.activity.header.build=Construction
+pipelinedetail.activity.header.commit=Consigne
+pipelinedetail.activity.header.completed=Termin\u00e9
+pipelinedetail.activity.header.duration=Dur\u00e9e
+pipelinedetail.activity.header.message=Message
+pipelinedetail.activity.header.status=\u00c9tat
+pipelinedetail.branches.header.branch=Branche
+pipelinedetail.branches.header.commit=Consigne
+pipelinedetail.branches.header.completed=Termin\u00e9
+pipelinedetail.branches.header.health=Sant\u00e9
+pipelinedetail.branches.header.message=Dernier message
+pipelinedetail.branches.header.status=\u00c9tat
+pipelinedetail.common.tab.activity=Activit\u00e9
+pipelinedetail.common.tab.branches=Branches
+pipelinedetail.common.tab.pullrequests=Demandes de tirage
+pipelinedetail.pullrequests.header.author=Auteur
+pipelinedetail.pullrequests.header.build=PR
+pipelinedetail.pullrequests.header.completed=Termin\u00e9
+pipelinedetail.pullrequests.header.status=\u00c9tat
+pipelinedetail.pullrequests.header.summary=Sommaire
+#run details
+rundetail.artifacts.button.download=T\u00e9l\u00e9charger l'artefact
+rundetail.artifacts.button.downloadAll.text=Tout t\u00e9l\u00e9charger
+rundetail.artifacts.button.downloadAll.title=T\u00e9l\u00e9charger tous les artefacts comme zip
+rundetail.artifacts.header.name=Nom
+rundetail.artifacts.header.size=Taille
+rundetail.artifacts.limit=Seuls les 100 premiers artefacts sont affich\u00e9s
+
+rundetail.changes.header.author=Auteur
+rundetail.changes.header.commit=Consigne
+rundetail.changes.header.date=Date
+rundetail.changes.header.message=Message
+rundetail.header.branch=Branche
+rundetail.header.changes.count={0} changements
+rundetail.header.changes.names=Changements par {0}
+rundetail.header.changes.none=Aucuns changements
+rundetail.header.commit=Consigne
+rundetail.header.tab.artifacts=Artefacts
+rundetail.header.tab.changes=Changements
+rundetail.header.tab.pipeline=Pipeline
+rundetail.header.tab.tests=Tests
+rundetail.pipeline.logs=Historique
```

Comment:
Bad translation if you really want to provide one. I think everybody is fine with "logs"